webv2/config/config_service.go

```diff
@@ -0,0 +1,93 @@
+package config
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/cloudspannerecosystem/harbourbridge/webv2/session"
+	"github.com/cloudspannerecosystem/harbourbridge/webv2/shared"
+)
+
+var configFilePath string = "./webv2/config.json"
+
+func GetSpannerConfig() (Config, error) {
+	var c Config
+	content, err := ioutil.ReadFile(configFilePath)
+	if err != nil {
+		log.Println(err)
+		return c, err
+	}
+
+	err = json.Unmarshal(content, &c)
+	if err != nil {
+		log.Println(err)
+		return c, err
+	}
+	return c, nil
+}
+
+//  Gets configuration from environment variables and saves it in config file.
+func LoadConfigFromEnv() {
+	var c Config
+	c.GCPProjectID = os.Getenv("GCPProjectID")
+	c.SpannerInstanceID = os.Getenv("SpannerInstanceID")
+
+	if c.GCPProjectID == "" || c.SpannerInstanceID == "" {
+		log.Println("warning : please set GCPProjectID and SpannerInstanceID as environment variables")
```

Comment:
nit: please set the environment variables - GCPProjectID and SpannerInstanceID ?

Reply:
updated the message.